### ISSUE TITLE
RAC: setting aria-label on MenuItem not working

### ISSUE THREAD

**Issue by hebergonza719** [adobe/react-spectrum#5795]
### Provide a general summary of the issue here

Setting `aria-label` on `MenuItem` doesn't apply the attribute. Property isn't displayed on the DOM.

### 🤔 Expected Behavior?

`aria-label` attribute should be applied on the element.

### 😯 Current Behavior

`aria-label` attribute isn't applied on the element.

### 🖥️ Steps to Reproduce

https://stackblitz.com/edit/stackblitz-starters-7a36ep?file=src%2FApp.tsx

### Version

"react-aria-components": "1.0.0"

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

macOS 13.6.4

**Comment by LFDanLu** [comment]
Looks like we forgot to pass the `aria-label` to the hooks for ListBoxItem and MenuItem, opened a PR for this. Will have to audit the rest of the components for parity. Thanks for reporting this!
